web-app/src/components/common/Input/index.js

```diff
@@ -109,22 +115,37 @@ const Input = props => {
       );
   }
 
+  const checkType = () => {
+    let styles = [];
+    styles.push(type);
+    if (type == 'checkbox') {
```

Comment:
I'd use === for type plus value if you see no issue

Reply:
updated to triple equals.